Styles/Classes/Extensions/NSAttributedString+Mutable.swift

```diff
@@ -0,0 +1,9 @@
+//  Copyright © 2018 Inloop, s.r.o. All rights reserved.
+
+import Foundation
+
+extension NSAttributedString {
+    func mutable() -> NSMutableAttributedString {
+        return mutableCopy() as! NSMutableAttributedString
```

Comment:
How about optional cast ?

Reply:
naah. We know that NSAttributedString's mutable variant is NSMutableAttributedString.